mewarpx/mewarpx/diags_store/diag_base.py

```diff
@@ -169,14 +169,16 @@ def __init__(self, diag_steps, preset_string='default',
         # (warp.top.steptime). Not sure what issues we'll hit just using time
         # here.
         self.prev_time = time.time()
+        self.start_time = self.prev_time
         self.prev_step = mwxrun.get_it()
         self.defaults_dict = {
             'default': "Step #{step:6d}; {nplive:8d} particles",
             'perfdebug': ("Step #{step:6d}; {nplive:8d} particles "
                           "{npperspecies} "
                           "{wall_time:6.1f} s wall time; "
                           "{step_rate:4.2f} steps/s; "
-                          "{particle_step_rate:4.2f} particle*steps/s"),
+                          "{particle_step_rate:4.2f} particle*steps/s; "
+                          "{particle_steps_total} particle*steps total"),
```

Comment:
I don't know that printing out the total particle pushes done so far is a really helpful output (it is a measure of how much work has been done, not how efficiently that work was done). Maybe printing the total particle step rate would be more informative?